.github/workflows/discord-release-notify.yml

```
@@ -0,0 +1,80 @@
+          if [ -z "${DISCORD_WEBHOOK_URL}" ]; then
+            echo "::error::DISCORD_WEBHOOK_URL secret is not set"
+            exit 1
+          fi
+
+          # Discord embed description max is 4096 chars; trim with a notice if longer.
+          BODY="${RELEASE_BODY:-_No release notes provided._}"
+          MAX=3900
+          if [ "${#BODY}" -gt "$MAX" ]; then
+            BODY="${BODY:0:$MAX}
+
+… [truncated — see [full release notes](${RELEASE_URL}))"
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)